Makefile

```diff
@@ -77,7 +77,7 @@ run-test-image-pr: # Suppress output as this contains sensitive information
 	-e GIT_REPO_SLUG=$(TRAVIS_REPO_SLUG) \
 	-e GIT_PULL_NUMBER=$(TRAVIS_PULL_REQUEST) \
 	-e CYPRESS_RBAC_TEST=$(CYPRESS_RBAC_TEST) \
-	-e CYPRESS_TEST_MODE=functional \
+	-e CYPRESS_TEST_MODE=smoke \
```

Comment:
I don't think this is used any more. Should we delete from this branch? I think only older branches using Travis need this.
![image](https://user-images.githubusercontent.com/42188127/134725780-3c8b019d-7370-4852-bb34-19b1dccc1d35.png)

Reply:
kk will make the change

Reply:
I meant to remove run-test-image-pr, not run-test-image actually. The latter is still referenced from the README.

Reply:
Ahh kk let me bring it back